test/statetest/statetest_loader.cpp

```diff
@@ -181,10 +195,25 @@ state::BlockInfo from_json<state::BlockInfo>(const json::json& j)
         }
     }
 
+    std::vector<state::Ommer> ommers;
+    if (const auto ommers_it = j.find("ommers"); ommers_it != j.end())
+    {
+        for (const auto& ommer : *ommers_it)
+        {
+            ommers.push_back(
+                {from_json<evmc::address>(ommer.at("address")), ommer.at("delta").get<uint8_t>()});
```

Comment:
This will also check if the test has valid number in `delta`
```suggestion
                {from_json<evmc::address>(ommer.at("address")), from_json<uint8_t>(ommer.at("delta"))});
```

Reply:
The `from_json<uint8_t>()` is not implemented, but by convention it would expect values in hex string. I also changed it to `uint32_t` but I'm thinking about using `int` because all difficulty calculations are done with signed integers.

Reply:
Keeping it as `uint32_t` because the reward is unsigned also.